samples/bluetooth/peripheral_with_multiple_identities/src/main.c

```diff
@@ -124,8 +124,8 @@ static int setup_advertiser(uint8_t id_adv)
 	/* Initialize the parameters for each connecable advertiser. */
 	struct bt_le_adv_param adv_param =
 		BT_LE_ADV_PARAM_INIT(BT_LE_ADV_OPT_CONNECTABLE,
-				     BT_GAP_ADV_SLOW_INT_MIN,
-				     BT_GAP_ADV_SLOW_INT_MAX,
+				     800, /* Minimum Advertising Interval (N * 0.625 milliseconds)  */
```

Comment:
My apologies for not making these changes. Looks good! We could make 800 a local #define ADV_INTERVAL 800. But that is really more of a nitpick :)